src/libs/OptionsListUtils.js

```diff
@@ -1453,6 +1453,7 @@ function getIOUConfirmationOptionsFromPayeePersonalDetail(personalDetail, amount
 function getIOUConfirmationOptionsFromParticipants(participants, amountText) {
     return _.map(participants, (participant) => ({
         ...participant,
+        text: participant.text || Localize.translateLocal('common.hidden'),
```

Comment:
Why this seems to be only needed for Native?

Reply:
Ah I see. But the suggested solution does not fix the bug. I think the `DisplayNames` component on itself is inconsistent as the `shouldUseFullTitle` prop is only available on non-native.

Reply:
Can we add the `Hidden` as a default in DisplayNames instead? I think that would make more sense

Reply:
Yes, that seems safer as OptionRow is a generic component where `Hidden` may not be wanted